CONTRIBUTING.md

```diff
@@ -313,3 +315,11 @@ Deployments are conducted automatically for lower environments and manually for
 ### Secrets and Environment Variables
 
 Both secrets and environment variables are managed by Defra tooling, speak with the engineering team to be briefed on this.
+
+## Demo applications
+
+### DEFRA form test
+
```

Comment:
Can we add a bit more context to explain to contributors why they might need or care about this?